src/rules/at-rule-no-unknown/index.js

```diff
@@ -83,8 +83,8 @@ export default function rule(primaryOption, secondaryOptions) {
             ruleName,
             result,
             node: warning.node,
-            line: warning.line,
-            column: warning.column
+            start: { line: warning.line, column: warning.column },
+            end: { line: warning.endLine, column: warning.endColumn },
```

Comment:
@ybiquitous is this a backwards compatible change?

Reply:
Good point. I tried testing the minimum version of `peerDependencies.stylelint` locally:

https://github.com/stylelint-scss/stylelint-scss/blob/efd565d2e0f1bd1c70c4242a94be200795dd5e51/package.json#L40-L42

```console
$ npm i stylelint@14.5.1
...

$ npx jest at-rule-no-unknown --no-verbose
 FAIL  src/rules/at-rule-no-unknown/__tests__/index.js
  ● scss/at-rule-no-unknown › reject › [ true ] › '@funciton floo ($n) {}\n' › no description

    expect(received).toMatchObject(expected)

    - Expected  - 2
    + Received  + 0

      Object {
        "column": 1,
    -   "endColumn": 10,
    -   "endLine": 1,
        "line": 1,
        "text": "Unexpected unknown at-rule \"@funciton\" (scss/at-rule-no-unknown)",
      }

      at Object.<anonymous> (node_modules/jest-preset-stylelint/getTestRule.js:88:33)
```

It seems that the backward compatibility keeps. 👍🏼 

If possible, I think it's better to update `peerDependencies.stylelint` to `^14.7.0`, but not required.